test/e2e/voting.feature

```diff
@@ -1,47 +1,23 @@
 Feature: Voting page
-  Scenario: should allow to view delegates and more on scroll
-    Given I'm logged in as "any account"
+  Scenario: should allow to view delegates and more on scroll and search them and vote for them
+    Given I'm logged in as "delegate candidate"
     And I wait 0.1 seconds
     When I go to "main/voting/"
     Then I should see 100 instances of "delegate row"
     When I scroll to the bottom of "delegate list"
     Then I should see 200 instances of "delegate row"
-
-  Scenario: should allow to search delegates
-    Given I'm logged in as "any account"
-    And I wait 0.1 seconds
-    When I go to "main/voting/"
     And I fill in "genesis_42" to "search" field
     Then I should see 1 instances of "delegate row"
-    And I clear "search" field
+    And I click "clean icon"
     And I fill in "doesntexist" to "search" field
     Then I should see 0 instances of "delegate row"
     And I should see text "No delegates found." in "empty message" element
-
-  @testnet
```

Comment:
There still should be a scenario that does a vote with `@testnet` because we had before the issue that all transactions worked with localhost but not with testntet/mainnet.